viadot/flows/sharepoint_to_adls.py

```diff
@@ -42,6 +42,7 @@ def __init__(
         if_exists: str = "replace",
         validate_df_dict: dict = None,
         timeout: int = 3600,
+        key_value_param: bool = False,
```

Comment:
It is already a similar parameter in BigQuery added: `set_prefect_kv`. What do you think about using the same name here?

Reply:
Done